vault/identity_store_util.go

```diff
@@ -591,6 +639,8 @@ func (i *IdentityStore) MemDBEntityByNameInTxn(txn *memdb.Txn, ctx context.Conte
 		return nil, fmt.Errorf("missing entity name")
 	}
 
+	entityName = i.sanitizeName(entityName)
```

Comment:
GH won't let me comment on line 637... Why does this func signature have ctx as the second parameter?

Reply:
Fixed.